test/AbstractAlgebra-test.jl

```diff
@@ -1,4 +1,4 @@
-using Random: MersenneTwister
+using Random: MersenneTwister, randsubseq
```

Comment:
Is this import still needed?

Reply:
Yes, I first didn't do it and all the tests failed on CI.